kodi_addon_checker/check_dependencies.py

```diff
@@ -27,6 +27,17 @@
                       'kodi.resource.font', 'kodi.inputstream', 'kodi.vfs', 'kodi.imagedecoder', 'xbmc.addon',
                       'xbmc.gui', 'xbmc.json', 'xbmc.metadata', 'xbmc.python', 'script.module.pil']
 
+extensions = {"kodi.gameclient": "kodi.binary.instance.game", "kodi.vfs": "kodi.binary.instance.vfs",
+              "xbmc.gui.skin": "xbmc.gui", "xbmc.metadata.scraper.albums": "xbmc.metadata",
```

Comment:
Moving every key value pair into a seperate line, would make it more easily to extend.